pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -587,8 +587,8 @@ func getFieldName(dataField data.Field, target *Query, metricTypeCount int) stri
 		return frameName
 	}
 	// todo, if field and pipelineAgg
```

Comment:
problem is, i have no idea what that line means :-(
i'll try to get back to it when we finish migrating everything, maybe it will make sense then. i hope that's ok.